demisto_sdk/commands/common/tests/readme_test.py

```diff
@@ -74,6 +74,41 @@ def test_are_modules_installed_for_verify_false_res(tmp_path):
     assert not readme_validator.are_modules_installed_for_verify(tmp_path)
 
 
+def test_relative_url_not_valid():
+    """
+    Given
+        - A README file with invalid relative urls in it.
+    When
+        - Run validate on README file
+    Then
+        - Ensure:
+            - Validation fails
+            - Both urls were caught correctly
+            - Valid url was not caught
+            - Image url was not caught
+    """
+    captured_output = io.StringIO()
+    sys.stdout = captured_output  # redirect stdout.
+    absolute_urls = ["https://www.good.co.il", "https://example.com", "https://github.com/demisto/content/blob/123",
+                     "/Packs/FeedOffice365/doc_files/test.png", "https://hreftesting.com"]
+    relative_urls = ["relative1.com", "www.relative2.com", "hreftesting.com", "www.hreftesting.com"]
+    readme_validator = ReadMeValidator(INVALID_MD)
+    result = readme_validator.verify_readme_relative_urls()
+    sys.stdout = sys.__stdout__  # reset stdout.
+    assert not result
+    output = captured_output.getvalue()
+    for url in absolute_urls:
+        assert url not in output
+
+    for url in relative_urls:
+        assert url in output
+
+    # no empty links found
+    assert '[ERROR]: /Users/ostolero/dev/demisto/demisto-sdk/demisto_sdk/tests/test_files/README-invalid.md: [RM112] ' \
+           '- Relative urls are not supported within README. If this is not a relative url, please add an https:// ' \
+           'prefix:\n. ' not in output
```

Comment:
```suggestion
    assert '[RM112]' not in output
```

Reply:
In this case, I'm checking no empty links were caught, so I actually need the end of the line.
I do see I accidentally added the local path so change this.